# Message storing raw 2D and 3D information about a single Object detection.
Header header
string id  # id of Object (not necessarily unique)
geometry_msgs/Pose pose  # pose of the marker in the camera frame
